templates/page_news_detail.html

```diff
@@ -30,15 +30,8 @@ <h1 class="mb-6 text-center">Introducing DjangoCon Africa 🌍 2024 🎉</h1>
       </div>
       <p>
         The African Django community is excited to announce the first DjangoCon
-        Africa event, taking place this year in Zanzibar, Tanzania, from 6th -
-        11th November 2023. The first event since the global pandemic postponed
-        our planning in 2020. We are thrilled to convene, support one another
-        and give back to the African Python web community now that travel and
-        health safety guidelines have normalized. DjangoCon Africa 2023 will be
-        held at the State University of Zanzibar where we have three meeting
-        rooms, one auditorium, 15 minutes away from food and activities, and
-        convenient access to students, professionals, and international
-        travelers.
+        Africa event, taking place this year in Arusha, Tanzania, from 11th -
+        15th August 2025. The second event will be held i Arusha, Tan.
```

Comment:
A minor typo in the last sentence 

**i** -> **in**

**Tan** -> **Tanzania**

The second event will be held **in** Arusha, Tan.

Reply:
Fixed, thanks for noticing.